mods/ts/rules/defaults.yaml

```diff
@@ -508,6 +508,30 @@
 		AirborneUpgrades: airborne
 	ReturnOnIdle:
 
+^AircraftHusk:
+	RenderVoxels:
+	WithVoxelBody:
+	BodyOrientation:
+		QuantizedFacings: 0
+		CameraPitch: 90
+	Aircraft:
+		RepairBuildings:
+		RearmBuildings:
+	Health:
+		HP: 280
+	Armor:
+		Type: Heavy
+	HiddenUnderFog:
+		Type: CenterPosition
+	AutoTargetIgnore:
+	ScriptTriggers:
+	Tooltip:
+		GenericName: Destroyed Aircraft
+	FallsToEarth:
+		Spins: True
```

Comment:
Still confusing since most values are upper case. Might want to check for this automatically and enforce it globally?